protocol/x/sending/keeper/transfer.go

```diff
@@ -215,6 +213,14 @@ func (k Keeper) SendFromModuleToAccount(
 	ctx sdk.Context,
 	msg *types.MsgSendFromModuleToAccount,
 ) (err error) {
-	// TODO(CORE-559): Implement this method.
-	return status.Errorf(codes.Unimplemented, "SendFromModuleToAccount not implemented")
+	if err = msg.ValidateBasic(); err != nil {
+		return err
+	}
+
+	return k.bankKeeper.SendCoins(
```

Comment:
Any particular reason to not use [SendCoinsFromModuleToAccount](https://github.com/dydxprotocol/cosmos-sdk/blob/4fadfe5a4606b6dc76644d377ed34420f3b80801/x/bank/keeper/keeper.go#L316)?